internal/service/cdss/cdss_cluster.go

```diff
@@ -132,9 +145,10 @@ func ResourceNcloudCDSSCluster() *schema.Resource {
 							ValidateDiagFunc: validation.ToDiagFunc(validation.IntBetween(3, 10)),
 						},
 						"storage_size": {
-							Type:     schema.TypeString,
-							Required: true,
-							ForceNew: true,
+							Type:             schema.TypeInt,
+							Required:         true,
+							ForceNew:         true,
+							ValidateDiagFunc: validation.ToDiagFunc(validation.IntBetween(100, 2000)),
```

Comment:
I think It could be a breaking change.
Could Integer type represent `storage_size = 2` likewise string type `storage_size = "2"`?

Reply:
ok then, good to go!